### ISSUE TITLE
Request to introduce "New search" method

### ISSUE THREAD

**Issue by itsmedakshesh** [Ylianst/MeshAgent#109]
Can you add a new feature where we can use multiple options in search to search a node in the mesh. like this "t:xyz; ip:xxx.xxx.xxx.xx " hope you get it.

One more example is as below:
![image](https://user-images.githubusercontent.com/78626086/134537454-4d6fa22d-f98b-47c5-9847-47db34f913d4.png)

**Comment by krayon007** [comment]
I was just discussing this use case with @Ylianst  Are you referring to a file system search?

**Comment by itsmedakshesh** [comment]
Yes, I meant to say here is, how about using multiple search tags to narrow down your search results. Like the one I mentioned in the screenshot. it shows that how we search for thread like "**is:issue**" then out of all the result, we again filter it out to "**is:open**".  I hope you understand what I am trying to say. Let me know I can try again if needed.. 👍🏻
